lib/src/rules/overridden_fields.dart

```diff
@@ -87,7 +87,7 @@ Iterable<InterfaceType> _findAllSupertypesAndMixins(
   return interfaces.where((i) => i != interface);
 }
 
-Iterable<InterfaceType> _findAllSupertypesInMixin(ClassElement classElement) {
+Iterable<InterfaceType> _findAllSupertypesInMixin(MixinElement classElement) {
```

Comment:
Consider renaming the parameter to match the new type.

Reply:
Done.  Thanks!